R/weight.R

```diff
@@ -12,21 +12,23 @@
 #' NULL. For the setter, x, modified accordingly. `weightVariables` returns
 #' the aliases (or names, according to `options(crunch.namekey.dataset)`),
 #' of the variables designated as weights.
-#' @aliases weightVariables
+#' @seealso [weightVariables()] [makeWeight()]
+#' @name weight
+#' @aliases is.weight<-
 #' @export
-weight <- function (x) {
+weight <- function(x) {
```

Comment:
Please undo the unnecessary linting that is inconsistent with our package style (space after the word "function")--it makes the diff look way bigger than it is.